src/Counter.php

```diff
@@ -41,16 +41,9 @@ final class Counter implements CounterInterface
 
     private int $ttlInSeconds = self::DEFAULT_TTL;
 
-    /**
-     * @var int Last increment time.
-     * In GCRA it's known as arrival time.
-     */
-    private int $lastIncrementTimeInMilliseconds;
```

Comment:
@romkatsu do you remember why it was a property?

Reply:
I do not remember. it seems that this is done to simplify the methods.